code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -68,7 +68,7 @@
 		return 0
 	var/title = "Sheet-Glass"
 	title += " ([src.amount] sheet\s left)"
-	switch(alert(title, "Would you like full tile glass or one direction?", "One Direction", "Full Window", "Cancel", null))
+	switch(input(title, "What would you like to construct?") in list("One Direction Window", "Full Window", "Fishbowl", "Fish Tank", "Wall Aquarium", "Cancel"))

```

Comment:
This is how the reinforced glass options were already handled, so I copied the style for the sake of simplicity. This can be changed at a later time with a more complete sweep of the code for similar stuff.